pkgs/applications/misc/kitty/default.nix

```diff
@@ -29,8 +29,8 @@ buildPythonApplication rec {
   outputs = [ "out" "terminfo" ];
 
   postPatch = ''
-    substituteInPlace kitty/utils.py \
```

Comment:
Could you move this into a patch file using `substituteAll`? Then issue like this will not happen in the future.

Reply:
Example: https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/video/gnome-mplayer/default.nix#L20

Reply:
Sure, will do in an additional commit and include the other replacement in it too.

Reply:
Hm, the other substitution in the Makefile is f[ishy too](https://hydra.nixos.org/build/82946154/log):
```PYTHONPATH=YTHONPATH:.. HOME=MPDIR/nowhere sphinx-build -M html "." "_build" -T  ```
The variable substituioons without curly brackets are invalid in makefiles. Will fix it too.